tensorflow/lite/tools/versioning/runtime_version.cc

```diff
@@ -124,7 +125,8 @@ std::string FindMinimumRuntimeVersionForOp(tflite::BuiltinOperator op_code,
            {{BuiltinOperator_FULLY_CONNECTED, 7}, "2.3.0"},
            {{BuiltinOperator_FULLY_CONNECTED, 8}, "2.3.0"},
            {{BuiltinOperator_FULLY_CONNECTED, 9}, "2.3.0"},
-           {{BuiltinOperator_FULLY_CONNECTED, 10}, "2.11.0"},
+           {{BuiltinOperator_FULLY_CONNECTED, 10}, "2.12.0"},
```

Comment:
@SaoirseARM 
Sorry for asking after approval. Was there any reason to change the previous version of CONV_2D and FULLY_CONNECTED? If not, I'll change it back myself on our inside process. (Same with line 75 of this file)